pages/common/xev.md

```diff
@@ -0,0 +1,20 @@
+# xev
+
+> Print contents of X events.
+> More information: <https://gitlab.freedesktop.org/xorg/app/xev>.
+
+- Monitor all occuring X events:
+
+`xev`
+
+- Monitor all X events of the root window instead of creating a new one:
+
+`xev --root`
+
+- Monitor all X events of a particular window:
+
+`xev -id {{window_id}}`
```

Comment:
One more inconsistency: Here you use one dash (`-id`), while the other examples use two (`--root`, `--event`). Could you normalize this to be consistent?

Reply:
According to the [man page](https://gitlab.freedesktop.org/xorg/app/xev/-/blob/master/man/xev.man) it should be `-root` and `-event`.

Reply:
Yes, it should. But that doesn't mean, it has to. I'm pretty sure, every X11 release ever has supported using two dashes. So it is a question of preference.

Reply:
It looks like `--id` does not work. I'm changing all to single `-`. 😄

Reply:
That's weird.